modules/ztp-monitoring-policy-deployment-progress.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-configuring-managed-clusters-policies.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-monitoring-policy-deployment-progress_{context}"]
+= Monitoring managed cluster policy deployment progress
+
+The ArgoCD pipeline uses `PolicyGenTemplate` CRs in Git to generate the {rh-rhacm} policies and then sync them to the hub cluster. You can monitor the progress of the managed cluster policy synchronization after the assisted service installs {product-title} on the managed cluster.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. The {cgu-operator-first} applies the configuration policies that are bound to the cluster.
++
+After the cluster installation is complete and the cluster becomes `Ready`, a `ClusterGroupUpgrade` CR corresponding to this cluster, with a list of ordered policies defined by the `ran.openshift.io/ztp-deploy-wave annotations`, is automatically created by the {cgu-operator}. The cluster's policies are applied in the order listed in `ClusterGroupUpgrade` CR.
++
+You can monitor the high-level progress of configuration policy reconciliation by using the following commands:
++
+[source,terminal]
+----
+$ export CLUSTER=<clusterName>
+----
++
+[source,terminal]
+----
+$ oc get clustergroupupgrades -n ztp-install $CLUSTER -o jsonpath='{.status.conditions[-1:]}' | jq
+----
++
+.Example output
+[source,terminal]
+----
+{
+  "lastTransitionTime": "2022-11-09T07:28:09Z",
+  "message": "The ClusterGroupUpgrade CR has upgrade policies that are still non compliant",
+  "reason": "InProgress",
+  "status": "True",
+  "type": "Progressing"
+}
+----
```

Comment:
@jc-rh is this status the same between 4.10 and 4.11? (specifically, is this output accurate for 4.10 TALM?)

Reply:
No, the Progressing condition is 4.12 only. BTW, the message for this condition is different in 4.12 as well:
https://github.com/openshift-kni/cluster-group-upgrades-operator/blob/a04dc548b4c240c0c7294b5cb3ae9441d9ca4d69/tests/kuttl/tests/upgrade-complete/02-assert.yaml#L26
@aireilly Switch to 4.10/4.11 version of this file as a reference for this PR.

Reply:
For 4.10, PTAL at: https://github.com/openshift/openshift-docs/pull/53026/commits/84b652c577d3686777b4ec19b6228342acb9f56f

I've review and a seperate PR fix for for 4.11

Reply:
4.11: https://github.com/openshift/openshift-docs/pull/53181

Reply:
@jc-rh ^^